crm_salesperson_planner/static/description/index.html

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is because you locally have an outdated pre-commit?

Reply:
I have updated pre-commit and this line has been removed. Is this what you meant?